src/Raven.Server/Web/System/TcpConnectionInfoHandler.cs

```diff
@@ -77,6 +79,12 @@ public async Task GetRemoteTaskTcp()
             var remoteTask = GetStringQueryString("remote-task");
             var database = GetStringQueryString("database");
             var verifyDatabase = GetBoolValueQueryString("verify-database", false);
+
+            if (ServerStore.IsPassive())
```

Comment:
This is not a good idea, note that this will mean that if the node is passive, we can't connect to it at all.
Including to be able to add to it to a cluster.
We need to check it only if this is for a particular db

Reply:
This endpoint is only for specific database, otherwise the endpoint /info/tcp will be used.
Note that in this endpoint, database is mandatory